src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Reflection/SignatureTypes.cs

```diff
@@ -237,6 +276,7 @@ public static void MakeSignatureArrayType()
             Assert.True(et.IsGenericParameter);
             Assert.False(et.IsGenericTypeParameter);
             Assert.True(et.IsGenericMethodParameter);
+            Assert.Throws<NotSupportedException>(() => et.IsValueType);
             Assert.Equal(5, et.GenericParameterPosition);
 
             TestSignatureTypeInvariants(t);
```

Comment:
Should we also have Assert's for IsValueType property in `MakeSignatureMdArrayType`, `MakeSignatureByRefType` and `MakeSignaturePointerType` below?

Reply:
Done